src/OAuthClient/HoldRequestClient.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace NYPL\HoldRequestResultConsumer\OAuthClient;
+
+use NYPL\HoldRequestResultConsumer\Model\DataModel\HoldRequest;
+use NYPL\Starter\APILogger;
+use NYPL\Starter\Config;
+
+class HoldRequestClient extends APIClient
+{
+    public static function getHoldRequestById($holdRequestId = '')
+    {
+        $url = Config::get('API_BASE_URL') . '/hold-requests/' . $holdRequestId;
```

Comment:
Perhaps it would be better to set the entire string as an environment parameter, like API_GET_HOLD_URL, instead of setting the entire string here?